en/messages.json

```diff
@@ -245,6 +245,32 @@
         "message": "Firefox Relay can now forward emails up to 25MB, including attachments.",
         "description": "DO NOT TRANSLATE \"Firefox Relay\"."
     },
+    "popupEndOfIntroPricingHeadline": {
+        "message": "Our intro pricing offer ends soon!"
```

Comment:
Does this have a character limit so it fits in one line? If so, what is the limit?

Reply:
This one can break off into a second line, it's just the countdown labels.